### PR TITLE
Add alexanderplatz document resources

### PR DESCRIPTION
**Pull request by pallix**
Branched from #1414

### CODE REVIEW

File: src/adhocracy_core/adhocracy_core/interfaces.py [hunk 1]
```diff
@@ -220,6 +220,14 @@ class ResourceMetadata(namedtuple('ResourceMetadata',
         Set addable content types, class heritage is honored
     """
 
+    def _add(self, **kw):
+        """Add elements to the value identified by key."""
+        last_modified = self
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/interfaces.py]: more precise:  "concatenate"  is whats happening here

File: src/adhocracy_core/adhocracy_core/interfaces.py [hunk 2]
```diff
@@ -220,6 +220,14 @@ class ResourceMetadata(namedtuple('ResourceMetadata',
...
+        for key, value in kw.items():
+            new_value = getattr(last_modified, key) + value
+            last_modified = last_modified._replace(**{key: new_value})
+        return last_modified
+
```
Comment (joka) [inline src/adhocracy_core/adhocracy_core/interfaces.py]: This method is also  usefull for other Named Tuple, not only Resource Metadata Tuple. 
A Named Tuple subtype would be nicer.

File: src/adhocracy_meinberlin/adhocracy_meinberlin/resources/kiezkassen.py [hunk 3]
```diff
@@ -24,17 +16,11 @@ class IProposalVersion(IItemVersion):
...
-                     ITitle,
-                     IDescription,
-                     adhocracy_meinberlin.sheets.kiezkassen.IProposal,
-                     IPoint,
-                     ICommentable,
-                     IRateable],
-    permission_create='edit_proposal',
+    extended_sheets=proposal.proposal_version_meta.extended_sheets +
+    [adhocracy_meinberlin.sheets.kiezkassen.IProposal,
+     IPoint],
```
Comment (joka) [inline src/adhocracy_meinberlin/adhocracy_meinberlin/resources/kiezkassen.py]: better use _add methdod instead here

**Comment by joka** [comment]
merge master is missing

**Comment by pallix** [comment]
Ready for second review and merge

**Pull request merged**